src/ws/cockpitcertificate.c

```diff
@@ -220,6 +220,8 @@ sscg_make_dummy_cert (const gchar *key_file,
     "--ca-file", ca_file,
     "--hostname", cn,
     "--organization", org,
+    "--subject-alt-name", "localhost",
+    "--subject-alt-name", "IP:127.0.0.1",
```

Comment:
This won't work. `sscg` will throw an error during the creation of the private CA certificate because it needs a complete `IP:127.0.0.1/255.255.255.255`. I *could* make it assume that it should have that restrictive submask by default, but I'd rather just conform to the specification.

Reply:
Oh, ok (sorry, I wasn't aware of that). I changed it accordingly then.